### DISCUSSION TITLE
Content & documentation

### DISCUSSION THREAD

**Discussion by juanfrank77** [nuejs/nue#30] [Q&A]
Hello!

I just found about this "nue" framework. And as a lover of minimalist things. I'm intrigued about the direction of this project. 

Don't know how early it is but I want to contribute with writing content and documentation since that is my strong suit.

Cheers.

**Comment by tipiirai** [comment]
WOW!! Super happy about this one! I'd like to talk about this online. Can you send email to "tipiirai" at gmail.com and we can continue from there? This is such a broad and open topic, quite hard to discuss here.

**Reply by juanfrank77** [reply]
Ok. Will do.
